### PR TITLE
Adds requirement on adding maps to the config in MAPS_AND_AWAY_MISSIONS.md

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request

There's been a surplus of ruin mappers lately, but a lot of them are unaware that they must be adding their file path to the config so server operators may toggle them on/off. I had to go through at least four separate PRs reminding people that this has to be done.

Let's add it to a centralized place regarding maps/mapping such that it's easier for maintainers to be aware that this requirement exists, as well as players who are scrounging the github attempting to figure out what they should do.

see the changes for yourself around here https://github.com/tgstation/tgstation/blob/1aac72b3e1798364073fa5af312ef4a5e00843fe/.github/guides/MAPS_AND_AWAY_MISSIONS.md

### CODE REVIEW

**Comment by san7890** [comment]
draft for a minute because i think i want to fold in some other parts of this MD

**Comment by san7890** [comment]
CC: @tgstation/maptainers @MrStonedOne am i forgetting something?

File: .github/guides/MAPS_AND_AWAY_MISSIONS.md [hunk 1]
```diff
@@ -52,7 +52,22 @@ For a much more comprehensive guide on UpdatePaths, please see the documentation
 
 Map files for away missions are located in the `_maps/RandomZLevels` directory. Each away mission includes it's own code definitions located in `/code/modules/awaymissions/mission_code`. These files must be included and compiled with the server beforehand otherwise the server will crash upon trying to load away missions that lack their code.
 
-To enable an away mission open `config/awaymissionconfig.txt` and uncomment one of the .dmm lines by removing the #. If more than one away mission is uncommented then the away mission loader will randomly select one of the enabled ones to load. We also support functionality for config-only away missions, which can be set up using the `config/away_missions` folder.
+Away missions are _disabled_ by default. Go to the file denoted in the [Configuration](#configuration) section and "untick" (remove the #) in order to enable it for loading. If more than one away mission is uncommented, the away mission loader will randomly select one of the enabled ones to load. We also support functionality for config-only away missions, which can be set up using the `config/away_missions` folder.
```
Comment (MrStonedOne) [inline .github/guides/MAPS_AND_AWAY_MISSIONS.md:55]:
```suggestion
<ins>Away missions are _disabled_ by default.</ins> Go to the file denoted in the [Configuration](#configuration) section and "untick" (remove the #) in order to enable it for loading. If more than one away mission is uncommented, the away mission loader will randomly select one of the enabled ones to load. We also support functionality for config-only away missions, which can be set up using the `config/away_missions` folder.
```
> <ins>Away missions are _disabled_ by default.</ins>

caveat: only works when viewed on github.

**Comment by Mothblocks** [comment]
This is good but making a lint for this would be extremely easy

**Comment by san7890** [comment]
> This is good but making a lint for this alongside it would be extremely easy

unsure where to even start on this though

**Comment by Mothblocks** [comment]
You don't have to but you could do something akin to:

`cat file1 file2 file3 file4 | grep /map_dmm_here`

**Pull request merged**